### DISCUSSION TITLE
file-name scheme for exporting recordings with motion

### DISCUSSION THREAD

**Discussion by sudo-joe** [blakeblackshear/frigate#4201] [General]
Hello forum,

is it possible to have Frigate named/exported recordings with motion according the following filename convention? 

- 2022-10-21--1602__Cam-yxz

Thank you!

**Comment by NickM-27** [comment]
Our API supports m3u8 and mp4 for whatever time range you want: https://docs.frigate.video/integrations/api#get-vodcamerastartstart-timestampendend-timestampindexm3u8 and can pull those recordings down.
